### ISSUE TITLE
Add convolutional autoencoder? 

### ISSUE THREAD

**Issue by oneengineer** [apache/incubator-mxnet#1549]

**Comment by piiswrong** [comment]
It's pretty easy. just connect convolution and deconvolution layers. 
You are welcome to contribute a example.

**Comment by mateuszr** [comment]
+1

**Comment by jmschrei** [comment]
The issue may be that he doesn't know how to do this and is looking for an example for guidance, so asking him to contribute an example is counterproductive.

**Comment by mateuszr** [comment]
You're most likey right, but anyway it could be a valid point to have some autoenconder example (apart from the mnist python, which could be good starting point, although probably too much bound to that concrete example). In general it is good to have a working example for most common use cases, as for many users this is a starting point to evaluate the toolkit. This is a good strategy for gaining the userbase (and other deep learning toolkits are doing it already).

**Comment by jmschrei** [comment]
I completely agree with having a plethora of examples for users. I'm just commenting that people asking for examples aren't always the best people to create them.

**Comment by FCInter** [comment]
I'm also looking forward to an implementation.
